### PR TITLE
docs(guide): update Dart user-input text to latest TS text

### PR DESCRIPTION
**Pull request by kwalrath**
I also made a few changes to the Dart sample.

This PR builds on #665.

### CODE REVIEW

**Comment by wardbell** [comment]
Is this ready to go?

File: public/docs/_examples/user-input/dart/lib/app_component.dart [hunk 1]
```diff
@@ -1,16 +1,14 @@
 // #docregion
-library user_input.app_component;
-
 import 'package:angular2/angular2.dart';
 import 'package:user_input/click_me_component.dart';
```
Comment (adaojunior) [inline public/docs/_examples/user-input/dart/lib/app_component.dart]: Maybe you should use relative imports to keep consistency with the other dart examples.

Comment (kwalrath) [inline public/docs/_examples/user-input/dart/lib/app_component.dart]: Good point.

**Comment by kwalrath** [comment]
About to merge this...

**Pull request closed without merging**